client/src/router/routes.js

```diff
@@ -87,6 +87,15 @@ const routes = [
         },
         props: true,
       },
+      {
+        name: "submission_view",
+        path: "/submission/view/:id",
+        component: () => import("src/pages/SubmissionView.vue"),
+        meta: {
+          requiresSubmissionAccess: true,
+        },
+        props: true,
```

Comment:
```suggestion
        name: "submission_review",
        path: "/submission/review/:id",
        component: () => import("src/pages/SubmissionReview.vue"),
        meta: {
          requiresSubmissionAccess: true,
        },
        props: true,
```
Ofc, this would necessitate renaming submissionview.vue as well.

Reply:
Well it's not the submission view so what do you suggest?

Reply:
How is it not the submission view?

Reply:
Since I recognize this can go either way, I'll rename occurrences of "view" to "review" as suggested.